Sources/ArcGISToolkit/Components/AR/TableTopSceneView.swift

```diff
@@ -149,6 +164,9 @@ public struct TableTopSceneView: View {
         // Add the visualization to the ARKit-managed node so that it tracks
         // changes in the plane anchor as plane estimation continues.
         node.addChildNode(planeNode)
+        
+        // Set help text when plane is visualized.
+        helpText = "Tap a surface to place the scene"
```

Comment:
Does the help text need to be localizable?

Reply:
Yes